### PR TITLE
Release 20160425

### PR DESCRIPTION
**Pull request by nzoschke**
- Restore build:create error event #572
- Log Events #570 
- Agent 0.68 #566
- Decouple certs from SSL #568
- Cherry-pick update-godeps d9bfaf2587748ef2afebf4cccdb5191266941038

## Release Playbook
- [x] Rebase against master
- [x] Release branch (20160425190157-release-20160425)
- [x] Pass CI (https://circleci.com/gh/convox/rack/746)
- [x] Code review
- [x] Merge into master
- [ ] Release master ()
- [ ] Pass CI ()
- [ ] Update staging rack
- [ ] Edit [Rack release record](https://github.com/convox/rack/releases) and/or update [docs](https://github.com/convox/convox.github.io)
- [ ] Publish release
- [ ] Release CLI

### CODE REVIEW

**Comment by nzoschke** [comment]
## QA

- [ ] test build failed event

File: api/dist/kernel.json [hunk 1]
```diff
@@ -28,6 +28,10 @@
       "Condition": "Development",
       "Value": { "Fn::If": [ "Autoscale", "true", "false" ] }
     },
+    "AwsAccount": {
```
Comment (nzoschke) [inline api/dist/kernel.json]: @ddollar is this intentional? I don't see where its referenced anywhere else

Comment (nzoschke) [inline api/dist/kernel.json]: n/m I see AWS_ACCOUNT in the Environment.

Comment (ddollar) [inline api/dist/kernel.json]: Ah I believe this is a remnant of something I was doing where I needed the account ID in Rack. Can be safely removed. Make sure it's out of the docker-compose.yml too

Comment (nzoschke) [inline api/dist/kernel.json]: Removed in c1b2f8f

**Comment by nzoschke** [comment]
Got this on CI everywhere:

> Uploading... ERROR: SignatureDoesNotMatch: The request signature we calculated does not match the signature you provided. Check your key and signing method.

Hypothesis is that this is a side effect of not doing Godeps properly. Pulling in d9bfaf2 with the hope it goes away,

**Comment by nzoschke** [comment]
It did go away with the better godeps commit

**Pull request merged**